### DISCUSSION TITLE
New Version 3.0.0

### DISCUSSION THREAD

**Discussion by sseworld** [sseworld/sseworld.github.io#1] [New Version]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

  📢 **Announce to your community** that we are releasing _new design of or documentation_!

### New Design Includes
- Uses the Docusorous
- Include Blog section

**Comment by ssewofficial** [comment]
Very Nice

**Reply by ssewofficial** [reply]
I am waiting for the new version
